homeassistant/components/matter/select.py

```diff
@@ -0,0 +1,186 @@
+            clusters.LaundryWasherMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterRefrigeratorAndTemperatureControlledCabinetMode",
+            translation_key="mode",
+        ),
+        entity_class=MatterModeSelectEntity,
+        required_attributes=(
+            clusters.RefrigeratorAndTemperatureControlledCabinetMode.Attributes.CurrentMode,
+            clusters.RefrigeratorAndTemperatureControlledCabinetMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterRvcRunMode",
+            translation_key="mode",
+        ),
+        entity_class=MatterModeSelectEntity,
+        required_attributes=(
+            clusters.RvcRunMode.Attributes.CurrentMode,
+            clusters.RvcRunMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterRvcCleanMode",
+            translation_key="mode",
+        ),
+        entity_class=MatterModeSelectEntity,
+        required_attributes=(
+            clusters.RvcCleanMode.Attributes.CurrentMode,
+            clusters.RvcCleanMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterDishwasherMode",
+            translation_key="mode",
+        ),
+        entity_class=MatterModeSelectEntity,
+        required_attributes=(
+            clusters.DishwasherMode.Attributes.CurrentMode,
+            clusters.DishwasherMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterMicrowaveOvenMode",
+            translation_key="mode",
+        ),
+        entity_class=MatterModeSelectEntity,
+        required_attributes=(
+            clusters.MicrowaveOvenMode.Attributes.CurrentMode,
+            clusters.MicrowaveOvenMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterEnergyEvseMode",
+            translation_key="mode",
+        ),
+        entity_class=MatterModeSelectEntity,
+        required_attributes=(
+            clusters.EnergyEvseMode.Attributes.CurrentMode,
+            clusters.EnergyEvseMode.Attributes.SupportedModes,
+        ),
+    ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=SelectEntityDescription(
+            key="MatterDeviceEnergyManagementMode",
+            translation_key="mode",
```

Comment:
Shoould all these clusters have the same translation key ? I could expect a device to have both energy mode clusters.

Reply:
Yes, it's just "mode" unless a description exists